modules/ding_event/ding_event.module

```diff
@@ -194,6 +194,10 @@ function ding_event_install_menu_position($op = 'install') {
  */
 function ding_event_views_pre_execute(&$view) {
   if ($view->name == 'ding_event' && in_array($view->current_display, array('ding_event_list'))) {
+    // Skip for term_view page. The BEF filter is not used here.
+    if ($view->args) {
+      return;
```

Comment:
Looks like we actually get a fatal error right now if we show the taxonomy term page for an event category? (see below)
![image](https://user-images.githubusercontent.com/5011234/70866103-a8a89580-1f65-11ea-9d37-1a002cbc1e19.png)

So this could maybe be considered an unrelated error, but since the term pages are not used much anyway we might as well fix it here.

I was wondering if there was a better way to handle this? It doesn't look the lines following the check (see below) are the reason for the error:

```
$view->build_info['query']->groupBy('node.nid');
$view->build_info['count_query']->groupBy('node.nid');
```

I think those modifications are actually important, but they are skipped if `$view->args` check returns TRUE and it's actually the modification in the next if-clause that causes the errors.

So I think that at least the check should be moved to the next if-clause and perhaps it would be better to check the `current_path()` and ensure that it's either "arrangementer" or the frontpage, since we only support the modification to the views on these pages.

Or maybe the `$view->args` are somehow directly related to the error? If so it may be the best way to check, but then I think a comment explaining the relation would be appropiate.

Reply:
Your absolutely right. I'll look into it right away.